README.md

```
@@ -1,381 +1,101 @@
+### 個別 PR の分析
 
-#### 開発環境用
+Claude Code に直接依頼：
 
-##### 方法1: APIエンドポイントを使用（推奨）
-
-サーバーが起動している状態で：
-
-```bash
-curl -X POST -H "X-Confirm-Clear: yes" http://localhost:8081/api/v1/clear
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)